weave/trace_server/clickhouse_trace_server_migrator.py

```diff
@@ -35,12 +37,16 @@ def apply_migrations(
         )
         if len(migrations_to_apply) == 0:
             logger.info(f"No migrations to apply to `{target_db}`")
+            if should_insert_costs(target_version):
```

Comment:
we should probably not do such migrations if we are migrating down

Reply:
I added a check to make sure we are not migrating down